### ISSUE TITLE
FindsByAndroidUIAutomator doesn't look for AccessibilityId

### ISSUE THREAD

**Issue by Icaroto** [appium/dotnet-client#308]
## Description

I'm trying to use the FindsByAndroidUIAutomator attribute to find a element by its AccessibilityId but the message I'm getting suggests it's trying to find by Id (which the element doesn't have). 
My 

## Environment

* Appium server: 1.10.1 and 4.0.0.5-beta nuget package
* Desktop OS/version used to run Appium if necessary: Mac
* Mobile platform/version under test: Android 8.0.0
* Real device or emulator/simulator: Real device (Galaxy s9)

## Details

This is the element

```
[FindsByAndroidUIAutomator(Accessibility = "Access")]
public IWebElement MyElement{ get; set; }
```

Then I just do a simple call like:

```
public void FindsByCustomOther()
{
    var test = MyElement.Displayed;
}
```

For the constructor I tried without success:
```
public MyClass(IWebDriver webDriver)
{
    PageFactory.InitElements(webDriver, this, new AppiumPageObjectMemberDecorator());
}
```

Do I need anything else to use it? As a workaround I created a custom FindsBy using the How.Custom from Selenium, but I'm puzzled on how to make this one work.

## Exception stacktraces

```
OpenQA.Selenium.NoSuchElementException: 'Couldn't locate an element by these strategies: Locator map: 
- native content: "By.Id: <OtherUserName>k__BackingField; " 
- html content: "ByIdOrName([<OtherUserName>k__BackingField]); " '
```

**Comment by lanzelot1989** [comment]
Does it work when using
```
[FindsByAndroidUIAutomator(Accessibility = "Access")]
public IMobileElement<AppiumWebElement> MyElement { get; set; }
```

**Comment by podupu** [comment]
I have the same issue it does not work with the below code 

> Does it work with?
> 
> ```
> [FindsByAndroidUIAutomator(Accessibility = "Access")]
> public IMobileElement<AppiumWebElement> MyElement { get; set; }
> ```

**Comment by Dor-bl** [comment]
@akinsolb FindsByAndroidUIAutomator was removed as part of the PageObjets deprecation, is that right?

**Comment by Dor-bl** [comment]
@Icaroto I see that when you posted this you used a beta version of this client.
If the issue is still relevant somehow can you try the latest stable [version](https://github.com/appium/dotnet-client/releases/tag/v4.4.0)?  although it's been 4 years now :(

**Comment by Dor-bl** [comment]
Closed as no response
